spec/support/platform_helpers.rb

```diff
@@ -231,3 +233,8 @@ def gce?
 rescue SocketError
   false
 end
+
+def choco_installed?
+  result = ShellHelpers.powershell_out("choco --version")
```

Comment:
This seems to work on OS X. I had some concerns about what `powershell_out` would do off of Windows.

```
2.3.1 :012 > ShellHelpers.powershell_out("choco --version").stderr
 => "sh: powershell.exe: command not found\n" 
```